docs/serverless-v2/01-02-serverless-v2.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Architecture
+---
+
+Serverless v2 replies on [Knative Serving](https://knative.dev/docs/serving/) for deploying and managing functions, and [Tekton](https://github.com/tektoncd/pipeline) as a pipeline for creating and running on-cluster processes. See how these and other resources process a lambda within a Kyma cluster:
```

Comment:
What means`running on-cluster processes`? Maybe just `pipeline for creating docker images`?